content/tracing/advanced_usage/_index.md

```diff
@@ -1505,10 +1529,16 @@ def hello():
 hello()
 ```
 
+[See our Python logging documentation][1] to ensure that the python log integration is properly configured and your python logs automatically parsed.
```

Comment:
I'm not entirely sure what you're trying to say here, but this doesn't parse. 😕

Reply:
Trying to say that it is important that the log are set with the `python` source as we do in the provided link.

Having the `python` source ensure that the logs are parsed and the trace id automatically extracted.

Reply:
```suggestion
[See our Python logging documentation][1] to ensure that the Python Log Integration is properly configured so that your Python logs are automatically parsed.
```